skywalking/trace/__init__.py

```diff
@@ -25,10 +25,13 @@
 
 
 class ID(object):
-    def __init__(self):
-        self.part1 = str(uuid.uuid1())
-        self.part2 = threading.get_ident()  # FIXME: thread ids are reused
-        self.part3 = int(time.time() * 1000) * 10000 + _id.next()
+    def __init__(self, string: str = None):
+        if string:
+            self.part1, self.part2, self.part3 = string.split('.')
+        else:
+            self.part1 = str(uuid.uuid1())
```

Comment:
If you have used UUID already, I would say, you only need one part. In java agent, the UUID is created once in the bootstrap stage.

Reply:
Thanks for pointing out, fixed now